### ISSUE TITLE
UI: Visualise all Schedules in a centralised place

### ISSUE THREAD

**Issue by fivenp** [kubeshop/testkube#3158]
**Describe the enhancement you'd like to see**
We want to allow users to see all their scheduled tests and test suites in one centralised place. 

Whether we want to do this on the Global Settings page or somewhere else – we'll find out....

![image](https://user-images.githubusercontent.com/132332/217502837-4ed84991-b7d4-4a80-8a2d-7af7cd281c24.png)

Figma mocks + flows: 
https://www.figma.com/file/asfUPepLHtyutWWR0pNGfK/Testkube-V2?node-id=1178%3A79377&t=aI6Eyn1qpXP6PMqq-4

**Additional context**

There are some things to consider here: 

1. The API does NOT have an endpoint yet to list all the schedules and return them consolidated (cc @vsukhin – we need to create a ticket for this) 
2. When we SAVE or EDIT a schedule – we will need send the post request to the related entity (there will be no new api endpoint)
3. This potentially requires us to have a global settings view (which we might not have yet -> #3159 )

**Comment by pavloburchak** [comment]
@fivenp @vsukhin any update on this one?

**Comment by vsukhin** [comment]
Low priority for now

**Comment by pavloburchak** [comment]
Blocked by API for now, moving to Blocked

**Comment by olensmar** [comment]
closing as this will be in commercial dashboard
